Converted CSS to style components

Simplystina opened the pull request.

Commit:
First commit

Commit:
Added top nav

Commit:
Added top navbar

Commit:
pulled main branch

Commit:
pulled main branch

LloydRivers committed:
Merge branch 'main' into cyber9ja

LloydRivers committed:
removed breaks

LloydRivers committed:
fix styling issues

Simplystina committed:
changed the css to style components

MorganBare commented on frontend/src/components/Navbar/Navbar.jsx:
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { NavContainer, NavTitle,NavButtonWrapper,NavButton,NavMenuIcon } from "./Navbar.Styled";
+//import {BiDotsVerticalRounded} from "re"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```
Could you please remove the FontAwesome library? We already have all the icons and everything we need in the assets folder.

MorganBare commented on frontend/src/components/Navbar/Navbar.jsx:
```diff
@@ -0,0 +1,19 @@
+    <NavContainer>
+        <NavTitle>Platform Launch</NavTitle>
+        <NavButtonWrapper>
+           <NavButton>+ Add New Task</NavButton>
```
We will need to add the '+' to a span or find a way to separate it from the 'Add new task' text as this text needs to be hidden on mobile and only the '+' will be shown.

MorganBare commented:
I left some comments for you but could you also please pull the main branch into this branch? The main is ahead of this branch.

MorganBare committed:
Pulled main into this branch and removed fontawesome library

MorganBare committed:
Added position: fixed to sidebar and a margin-left to the navbar so it aligns properly

MorganBare committed:
centered the add new task form

MorganBare committed:
Added a column container and added a display flex on that to position the three columns correctly. Also added flex-direction: row and a gap the individual columns

LloydRivers committed:
remove conole.log

LloydRivers committed:
Merge branch 'cyber9ja' of https://github.com/chingu-voyages/v40-geckos-team-12 into cyber9ja

LloydRivers committed:
Modal styling

I have moved a few things around so that the layout looks good. I added the function that will open the add task modal too.

LloydRivers committed:
removed fixed height

The icon need to be aligned now.

On the parent element, add align items centre

LloydRivers committed:
removed margin

LloydRivers committed:
follow style of other modal

LloydRivers committed:
follow style of other modal

LloydRivers committed:
um...

I don't remember doing this change, but it is obvious what it does.

LloydRivers committed:
create export AddTaskModal

So, I created a new component so that when the form modal

LloydRivers committed:
NO MORE FIXED POSITION!

Please don't add this back.

LloydRivers committed:
styling

Changed the styling to match the modal we have in the app already

LloydRivers committed:
align kebab centre

LloydRivers committed:
correct main container BG

LloydRivers committed:
dark mode

These commits are to fix the colors of the fonts in dark and light mode.

Also, there were  styled components made that didnt even have a color property on them, so how you expect them to toggle?

LloydRivers committed:
move the slider

I have styled and moved the slider to where it needs to go.
